static/templates/draft.handlebars

```diff
@@ -0,0 +1,42 @@
+<div class="draft-row" data-draft-id="{{draft_id}}">
+  <div class="draft-info-box">
+    {{#if is_stream}}
+    <div class="message_header message_header_stream">
+      <div class="message-header-contents">
+        <div class="message_label_clickable stream_label"
+          style="background: {{stream_color}}; border-left-color: {{stream_color}};">
+          {{stream}}
+        </div>
+
+        <span class="stream_topic">
+          <div class="message_label_clickable narrows_by_subject">
+            {{topic}}
+          </div>
+        </span>
+      </div>
+    </div>
+    {{else}}
+    <div class="message_header message_header_private_message dark_background">
+      <div class="message-header-contents">
+        <div class="message_label_clickable stream_label">
+          You and {{recipients}}
```

Comment:
This should be tagged for translation just like in `static/templates/recipient_row.handlebars`